grails-app/domain/org/pih/warehouse/shipping/ShipmentItem.groovy

```diff
@@ -73,6 +74,8 @@ class ShipmentItem implements Comparable, Serializable {
             "unavailableQuantityPicked",
             "paymentTerm",
             "quantityInvoiced",
+            "orderItem",
+            "uom",
```

Comment:
I don't have a strong opinion on that, but I can see that in some places we use `uom` and in some places we use `unitOfMeasure` - I think it would be good to align on one version, as more PRs related to that will come.

Reply:
For now, I'd say that `unitOfMeasure` is more relevant, because it's currently used in `OrderItem` and `Product` domain.